src/LocalCoords.cpp

```diff
@@ -77,6 +77,16 @@ int LocalCoords::getLatticeY() const {
 
 
 /**
+ * @brief Return the third index of the Lattice cell within which this
+ *        LocalCoords resides.
+ * @return the third Lattice cell index
+ */
+int LocalCoords::getLatticeZ() const {
+  return _lattice_z;
```

Comment:
Seeing that a 3D lattice coordinate is now built into `LocalCoords` makes me even more strongly believe that we should only accept 3D lists of lattice universes.